blogpost-drools-model-compiler.md

```diff
@@ -0,0 +1,116 @@
+
+## What is the Executable Model?
+
+The Executable Model is a new way to execute business rules in Drools which provides a much faster startup time. You can check out the details in the [Drools Documentation](https://docs.jboss.org/drools/release/7.55.0.Final/drools-docs/html_single/index.html) or in other blog posts such as [Mario's](https://blog.kie.org/2021/06/profiling-and-optimizing-executable-model-generation.html).
+
+KJARs built with the `kie-maven-plugin` have the Executable Model enabled since 7.33.0.Final by default and it's the main technology underneath [Kogito](https://kogito.kie.org). 
+With the "Executable Model Compiler", a module you can find in the  `drools-model-compiler` directory, DRL files are transformed into a Java DSL.
+
+## How to start contributing
+
+Drools is a really big open source project, and finding the best way to contribute to it might not be easy.
+Luckily, the Executable Model Compiler is a good way to start, for various reasons:
+
+* It's a fairly new project (as today it's been more or less three years since the inception)
+* It doesn't require deep understanding of the Drools' internal algorithm, PHREAK
+* There always is a former counterpart to verify the code against 
+
+Regarding the third point, we want to make sure that Drools is behaving in the exact same way while using the former runtime (also called "DRL") and the new one (called `PatternDSL`).
+
+## Contributing: showing a problem 
+
+Imagine that you're interested in contributing to Drools, what should you do when you find a problem and you think it's related to the Executable Model? 
+
+First we should understand how to recognise whether it's an error with Drools or specifically in the Executable Model. 
+
+To do that, we need to create the smallest piece of code that shows the problem: this is what we called a "bug reproducer" (also just "reproducer"). 
+If you provide a bug report to [Zulip](https://blog.kie.org/2020/04/new-community-channels-on-zulip-chat.html), mailing lists or [Stack Overflow](https://drools.org/community/getHelp.html)
+the team will ask you to create such reproducer. There are two ways to do it:
+
+1) If you're familiar with the [Drools GitHub repository](https://github.com/kiegroup/drools) you can write the test directly in your own fork of the original repository and create a PR against it. 
+   This is probably the best way to proceed, as it allows all the Drools' developers to check the problem faster
+2) Create another separate project that shows the problem
+
+## Method #1: create a reproducer in drools.git
+
+Start by building the Drools project reading the [Drools Source Code](https://drools.org/code/sourceCode.html) page.
+
+You can either decide to build it using [droolsjbpm-build-bootstrap](https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/master/README.md) or building only the [Drools](https://github.com/kiegroup/drools) module with `mvn clean install`. The second one is definitely faster.
```

Comment:
I think mentioning `droolsjbpm-build-bootstrap` could be misleading for the user: in general `drools.git` should be enough, then we can add a link to the other repo just as additional note like "Drools project is part of Kie community so there are many other repos, if you want to have more information start from this that is the entry point and check the contributing guide"